### PR TITLE
Build the plugin and extract from correct dir.

### PR DESCRIPTION
**Pull request by AnthonyLedesma**
### Description

The deploy to WordPress workflow is now fixed, and can be run manually at any time by heading to https://github.com/godaddy-wordpress/coblocks/actions/workflows/deploy-wordpress.yml and clicking 'Run Worklow' in the top right. You will just need to enter the version being deployed (should match the plugin header in `master`).

A second workflow was added that will run whenever a tag is committed to GitHub. The tag is then built and deployed to WordPress.org SVN tags. The tag _will not_ be copied to trunk. The only time trunk/tags are both updated is when the `Deploy / WordPress` workflow mentioned above is manually triggered.

### Screenshots
image

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
Ran the workflow on GitHub actions.

### Acceptance criteria
- Resolve failing deploy workflow.
- Introduce a new workflow that deploys the tag to the appropriate WordPress.org tags directory.

### Checklist:
- [x] My code is tested
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by EvanHerman** [comment]
I believe @aledesma-godaddy is fixing the e2e errors in https://github.com/godaddy-wordpress/coblocks/pull/2501

File: .distinclude [hunk 1]
```diff
@@ -1,17 +1,21 @@
 # List all files and directories to be deployed with the Go thene.
```
Comment (olafleur-godaddy) [inline .distinclude:1]: Is it a copy-paste error of talking about Go or it's talking about the Go install that CoBlocks is using?
Also, it should say `theme`

**Review by olafleur-godaddy — APPROVED** [review]
Looking good!

**Comment by olafleur-godaddy** [comment]
> I believe @aledesma-godaddy is fixing the e2e errors in #2501

Ok, great! The quicker we can get to green, the better ;)

File: .distinclude (hunk 1 above)
Comment (EvanHerman) [reply .distinclude]: Yeah, you are right. This should just be "deployed with CoBlocks". I've updated that, good catch 👍

**Review by AnthonyLedesma — COMMENTED** [review]
I created the PR so I can not put approval on there but this looks good. Glad to see tests passing. In what ways did you test this? I assume you may have set up a tag deployment. Did you happen to try any other part of the deployment?
@EvanHerman

**Comment by EvanHerman** [comment]
@AnthonyLedesma I tested it by running steps except the actual svn commit. When I was happy with where it was I re-added that last command in there

**Pull request merged**